scripts/treedb_vacuum_m0_summarize.py

```
@@ -0,0 +1,197 @@
+
+
+def parse_benchmark(path: Path) -> dict[str, float]:
+    rows = []
+    for line in path.read_text().splitlines():
+        if not line.startswith("Benchmark"):
+            continue
+        fields = line.split()
+        if len(fields) < 4 or len(fields[2:]) % 2:
+            raise ValueError(f"malformed benchmark row in {path}: {line}")
+        metrics = {}
+        for value, unit in zip(fields[2::2], fields[3::2]):
```

Ruff (0.15.21)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)